lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -162,7 +172,7 @@ class GoogleCloudOutput < BufferedOutput
     attr_reader :running_on_managed_vm
     attr_reader :gae_backend_name
     attr_reader :gae_backend_version
-    attr_reader :service_name
+    attr_reader :resource_type
```

Comment:
Ah, I thought `merge_into_self` was a Ruby method. We can do this instead, then:
```
module Google
  module Protobuf
    # Add Map::merge!.
    class Map
      def merge!(hash)
        hash.each { |k, v| self[k] = v }
        self
      end
    end
  end
end
```